simulator/src/main/java/com/hedera/block/simulator/generator/Utils.java

```diff
@@ -40,4 +40,20 @@ public static byte[] readGzFile(Path filePath) throws IOException {
             return gzipInputStream.readAllBytes();
         }
     }
+
+    /**
+     * Read a file and return the content as a byte array.
+     *
+     * @param filePath Path to the file
+     * @return byte array of the content of the file
```

Comment:
A quick general suggestion I think is a good idea in the `@return` part of the javadoc to also say what would happen if something. For example: `@return byte array of the content of the file, or null if file extension is not supported`. During development this is very helpful and also is a good explanation of what the method should achieve and if there is a bug, by knowing what we should be doing, we could find it easy.

Consider this practice everywhere, I think it is good.

Reply:
✅  Done